NSS-Day-Cohort-55/automated-world-granny-mashinini-fires
# added fonts to CSS and country images

# Description

## Type of change


# Testing Instructions
check to make sure images folder with country images shows up, make sure fonts change throughout

# Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
